<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderLibraryWidget</class>
 <widget class="QDockWidget" name="ShaderLibraryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shader Library</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="ShaderLibTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Programs_tab">
       <attribute name="title">
        <string>Programs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>246</width>
             <height>219</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTreeView" name="ProgramTreeView">
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectItems</enum>
              </property>
              <property name="itemsExpandable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="ProgramAddButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/TimelineToolbar/Add16.png</normaloff>:/TimelineToolbar/Add16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ProgramRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/TimelineToolbar/Delete16.png</normaloff>:/TimelineToolbar/Delete16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ProgramEditButton">
            <property name="text">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Shaders_tab">
       <attribute name="title">
        <string>Shaders</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>246</width>
             <height>219</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTreeView" name="ShaderTreeView"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="ShaderAddButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/TimelineToolbar/Add16.png</normaloff>:/TimelineToolbar/Add16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ShaderRemoveButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../Resources/Resources.qrc">
              <normaloff>:/TimelineToolbar/Delete16.png</normaloff>:/TimelineToolbar/Delete16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="ShaderEditorButton">
            <property name="text">
             <string>E</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
